main/src/main/java/com/google/android/apps/muzei/widget/AppWidgetUpdateTask.kt

```diff
@@ -179,9 +179,34 @@ private fun Bitmap.scale(widgetWidth: Int, widgetHeight: Int): Bitmap? {
             widgetWidth == 0 || widgetHeight == 0) {
         return null
     }
+    val widgetIsLandscape = widgetWidth >= widgetHeight
+    // Crop image first here for highest quality
```

Comment:
Move this center cropping to its own method, then `createRemoteViews` can use `val scaledImage = image.centerCrop(widgetWidth, widgetHeight).scale(widgetWidth, widgetHeight)`